packages/esm-shr-app/src/index.ts

```diff
@@ -35,6 +42,10 @@ export const ReferralsDashboardLink = getSyncLifecycle(
   options,
 );
 
+export const shrSummaryDashboardLink = getFeatureFlag('shr-summary')
+  ? getSyncLifecycle(createDashboardLink({ ...shrSummaryDashboardMeta, moduleName }), options)
+  : undefined;
```

Comment:
This isn't correct
```suggestion
export const shrSummaryDashboardLink = getSyncLifecycle(createDashboardLink({ ...shrSummaryDashboardMeta, moduleName }), options)
```

for this under routes.json we can have to toggle the feature flag
```json
{
      "name": "patient-chart-shr-summary-dashboard-link",
      "component": "shrSummaryDashboardLink",
      "slot": "patient-chart-dashboard-slot",
      "meta": {
        "columns": 1,
        "columnSpan": 1,
        "slot": "patient-chart-shr-summary-dashboard-slot",
        "path": "SHR Portal"
      },
      "featureFlag": "healthInformationExchange"
    },
```